src/pages/using-spark/components/button.mdx

```diff
@@ -19,72 +20,32 @@ Well written button text clearly communicates what action will occur when the us
 
 ### Usage
 
-Use Buttons to represent what type of actions the users can take on a page and to create a visual hierarchy between different calls to action.
-
-> As a general rule, use no more than two primary call-to-action buttons ([Primary Button](#primary-button)) per page.
-
-Examples of Button usage:
-- To trigger action.
-- To submit a form.
-- To begin a new task.
-- To navigate to the next step in a process.
-- To confirm completion or to exit out of a flow.
+Use buttons to communicate the different actions users can take and how they can interact with the page.
 
-If the action will take the user to a new page or to an external site, consider using a [Link](/using-spark/components/link) instead.
+> As a general rule, use one main call to action ([Primary Button](#primary-button)) per screen.
```

Comment:
This is picky, but I think "screen" is better as "page" here. Screen would imply the visual box that the user sees, not the whole page in general.